internal/services/pubkey_service.go

```diff
@@ -4,11 +4,14 @@ import (
 	"net/http"
 
 	"github.com/RHEnVision/provisioning-backend/internal/dao"
+	"github.com/RHEnVision/provisioning-backend/internal/db"
 	"github.com/RHEnVision/provisioning-backend/internal/payloads"
 	"github.com/go-chi/render"
 	"github.com/pkg/errors"
 )
 
+var FingerprintReadOnlyError = errors.New("fingerprint is a read-only field")
```

Comment:
Looks like a dead code.
I was thinking we can directly [filter it out](https://github.com/go-chi/chi/blob/9db25e5b381fbd6350583b857e74ecaffe355819/_examples/rest/main.go#L328) in the request payload, wouldn't it be a good strategy?
That way we would ignore the field completly

Reply:
Absolutely makes sense, this is exactly what Bind/Render functions are for! It was just one-liner. Dropped this unused line too.